qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -88,6 +88,44 @@ def pass_manager(self, pass_manager_config, optimization_level=None) -> PassMana
                     pass_manager_config.unitary_synthesis_plugin_config,
                     pass_manager_config.hls_config,
                 )
+        elif optimization_level in {1, 2}:
```

Comment:
Incredibly minor: there's no reason to use a `set` here - `optimization_level in (1, 2)` would technically be more efficient.  Not important at all, though.